src/edu/washington/escience/myria/tools/MyriaConfigurationParser.java

```diff
@@ -323,6 +325,46 @@ private static int getPort(final ConfigParser parser, final int nodeId)
 
   /**
    *
+   * @param nodeId the worker/master ID
+   * @return the port number
+   * @throws ConfigFileException if error occurred when getting the value
+   */
+  private static int getJVMPort(final ConfigParser parser, final int nodeId)
```

Comment:
Doesn't the JVM port only exist in your hacked OpenJDK version? What happens if a user specifies a nonzero value for this port on a standard JVM?

Reply:
See above.